airflow-core/src/airflow/ui/public/i18n/locales/de/dag.json

```
@@ -103,6 +108,7 @@
     "mappedTaskInstances_other": "Task Instanzen [{{count}}]",
     "overview": "Übersicht",
     "renderedTemplates": "Ausgefüllte Vorlagen",
+    "requiredActions": "Erforderliche Interaktionen",
```

@TJaniF / @m1racoli The tab text is quite long with a risk of adding a lin break - WDTY - is it okay to shorten the translation to required context? "Erforderliche" as adjective might be not needed?
```suggestion
    "requiredActions": "Interaktionen",
```

---

Hmm... wenn ich #53035 richtig lese ist das der tab unter dem HITL Aufgaben erscheinen, egal in welchem status. Ich denke ja, in dem Fall würde ich "erforderliche" auch streichen, das impliziert ja, dass da immer noch HITL Aufgaben warten.

Noch tricky da ein gutes Wort zu finden. "Mensch gebraucht" 😅 ist jetzt auch nicht so technisch. Ja, ich denke Interaktionen ist gut :)